src/Microsoft.DotNet.Build.Tasks/PackageFiles/executor/executor.cs

```diff
@@ -0,0 +1,227 @@
+﻿using System;
+using System.IO;
+using Newtonsoft.Json;
+using Newtonsoft.Json.Linq;
+using System.Collections.Generic;
+
+namespace Executor
```

Comment:
Should this be Microsoft.* instead? Also, why the name Executor? It seems to me that Parser fits more the purpose of this.